pkg/daemon/feature/feature.go

```diff
@@ -0,0 +1,51 @@
+package feature
```

Comment:
package should after copyright

Reply:
fixed, replace pkg/daemon/feature with pkg/feature.